apps/dashboard/src/components/primitives/field-editor/variable-popover/variable-popover.tsx

```diff
@@ -0,0 +1,257 @@
+            </div>
+          </div>
+        </div>
+        <div className="grid gap-1.5">
+          <div className="grid gap-2 p-2">
+            <FormItem>
+              <FormControl>
+                <div className="grid gap-1">
+                  <label className="text-text-sub text-label-xs">Variable name</label>
+                  <InputField size="fit" className="min-h-0">
+                    <Code2 className="text-text-sub h-4 w-4" />
+                    <Input value={name} onChange={(e) => handleNameChange(e.target.value)} className="h-7 text-sm" />
+                  </InputField>
+                </div>
+              </FormControl>
+            </FormItem>
+            <FormItem>
+              <FormControl>
+                <div className="grid gap-1">
+                  <label className="text-text-sub text-label-xs">Default value</label>
+                  <InputField size="fit" className="min-h-0">
+                    <Input
+                      value={defaultVal}
+                      onChange={(e) => handleDefaultValueChange(e.target.value)}
+                      className="h-7 text-sm"
+                    />
+                  </InputField>
+                </div>
+              </FormControl>
+            </FormItem>
+            <FormItem>
+              <FormControl>
+                <div className="flex items-center justify-between">
+                  <label className="text-text-sub text-label-xs">Show raw liquid</label>
+                  <Switch checked={showRawLiquid} onCheckedChange={setShowRawLiquid} className="scale-75" />
+                </div>
+              </FormControl>
+            </FormItem>
+            {showRawLiquid && (
+              <FormItem>
+                <FormControl>
+                  <div className="grid gap-1">
+                    <InputField size="fit" className="min-h-0">
+                      <Input
+                        value={formatLiquidVariable(name, defaultVal, transformers)}
+                        onChange={(e) => handleRawLiquidChange(e.target.value)}
+                        className="h-7 text-sm"
+                      />
+                    </InputField>
+                  </div>
+                </FormControl>
+              </FormItem>
+            )}
+          </div>
+
+          <Separator className="my-0" />
+
+          <div className="flex flex-col gap-1 p-2">
+            <FormItem>
+              <FormControl>
+                <div className="grid gap-1">
+                  <label className="text-text-sub text-label-xs">Modifiers</label>
+                  <Select
+                    value=""
+                    onValueChange={(value) => {
+                      if (value) {
+                        handleTransformerToggle(value);
+                        setSearchQuery('');
+                      }
+                    }}
+                  >
```

Comment:
the height should be the same of the other inputs

https://github.com/user-attachments/assets/48eb8260-06b1-4dca-9743-9454b424e1f4